tsconfig.spec.json

```diff
@@ -0,0 +1,18 @@
+{
+  "extends": "./tsconfig.json",
+  "include": ["./jest/testing-library__jest-dom.d.ts", "./jest/verifyComponent.d.ts", "**/*.spec.tsx"],
+  "exclude": [],
+  "compilerOptions": {
+    "declaration": false,
+    "declarationMap": false,
+    "noImplicitReturns": true,
+    "noImplicitThis": true,
+    "preserveConstEnums": true,
+    "allowUnreachableCode": false,
+    "noImplicitAny": false,
+    "strictNullChecks": false,
```

Comment:
I set these to be false for tests. I've had these on `true` in the past, but it is more overhead in tests... And it isn't actually catching anything useful for us - tests already catch things... These were essentially disabled before since we didn't have any custom config for spec files. The overall effect is more strict typing than before.